cli/pkg/release/http.go

```diff
@@ -44,6 +45,61 @@ func prepareHttpRequest(url string, parsedArgs *ReleaseParameters) (*http.Reques
 		}
 	}
 
+	for environment, signature := range parsedArgs.Signatures {
+		part, err := writer.CreateFormFile(fmt.Sprintf("signatures[%s]", environment), fmt.Sprintf("%s-signature", environment))
+		if err != nil {
+			writer.Close()
+			return nil, fmt.Errorf("error creating the form entry for environment %s with signature file %s, error: %w", environment, signature, err)
+		}
```

Comment:
if there's an error, then we don't need to close the writer
```suggestion
		part, err := writer.CreateFormFile(fmt.Sprintf("signatures[%s]", environment), fmt.Sprintf("%s-signature", environment))
		if err != nil {
			return nil, fmt.Errorf("error creating the form entry for environment %s with signature file %s, error: %w", environment, signature, err)
		}
		defer writer.Close()

```